serverless/cmd/sequence/main.go

```diff
@@ -51,22 +74,30 @@ func main() {
 	h := hasher.DefaultHasher
 	// init storage
 	var st *fs.Storage
-	if *create {
-		st, err = fs.Create(*storageDir, h.EmptyRoot())
-	} else {
-		var cpRaw []byte
-		cpRaw, err = fs.ReadCheckpoint(*storageDir)
-		if err != nil {
-			glog.Exitf("Failed to read log checkpoint: %q", err)
-		}
-		var cp *log.Checkpoint
-		if _, err := cp.Unmarshal(cpRaw); err != nil {
-			glog.Exitf("Failed to unmarshal checkpoint: %q", err)
-		}
-		st, err = fs.Load(*storageDir, cp)
+
+	cpRaw, err := fs.ReadCheckpoint(*storageDir)
+	if err != nil {
+		glog.Exitf("Failed to read log checkpoint: %q", err)
+	}
+
+	// Check signatures
+	v, err := note.NewVerifier(pubKey)
+	if err != nil {
+		glog.Exitf("Failed to instantiate Verifier: %q", err)
+	}
+	verifiers := note.VerifierList(v)
```

Comment:
inline if you like

Reply:
I'm not sure what you mean by this...

Reply:
sorry - I should've been a bit more verbose!
Since `verifiers` is only used on the line below, and there's no `err` returned in its construction, you could just squash this and the next line together:
```golang
vCp, err := note.Open(cpRaw, note.VerifierList(v))
```

Reply:
ack, thanks! Done